helm-charts/seldon-core-operator/templates/deployment_seldon-controller-manager.yaml

```diff
@@ -160,11 +160,16 @@ spec:
           readOnly: true
 {{- end }}
       hostNetwork: {{ .Values.hostNetwork }}
+      nodeSelector:
```

Comment:
It seems this would be passed as empty if not provided - it doesn't seem to error, but it does seem it may be better to ignore the field completely with a guard (similar to the values above)